notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-798_analysis.ipynb

```diff
@@ -0,0 +1,655 @@
+{
```

Comment:
<p>I believe you did some testing ultil you finally picked that specific slew (i_slew = 450). What about giving some context here on how did you get it?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/94/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
It is actually a more or less random choice for that day, for illustration purposes. I will clarify this.